### PR TITLE
Adds nice soft gloves.

### PR DESCRIPTION
**Pull request by GuyonBroadway**
:cl: Guyonbroadway
add: Clowns and mimes now start the game with soft gloves, when you click someone with help intent using these gloves it will set their intent to help.
add: You can also get an extra pair from the autodrobe with a coin.
/:cl:

Mostly to finally deal with those situations where some fuckwit is walking down a 1x1 maint tunnel and refuses to switch to help intent and is all like "**DUDE WTF!! MOVE!!**"

It will also be a godsend during blob rounds for the same reason.

### CODE REVIEW

**Comment by GuyonBroadway** [comment]
Yes I also tested this PR.

**Review by ghost — APPROVED** [review]
LGTM

**Comment by GuyonBroadway** [comment]
Refactoring code to use something better.

**Comment by GuyonBroadway** [comment]
Ok lets try again

File: hippiestation/code/modules/clothing/gloves/softgloves.dm [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	name = "soft gloves"
+	desc = "These gloves are very soft and feel nice to touch."
+
+/obj/item/clothing/gloves/color/white/soft/Touch(mob/living/carbon/human/target,proximity)
+	var/mob/M = loc
+	if(ishuman(target))
+		if(M.a_intent == INTENT_HELP && target.a_intent != INTENT_HELP)
+			target.a_intent = INTENT_HELP
+			target.hud_used.action_intent.icon_state = "[target.a_intent]" //Else we get your intent being one thing and your hud another.
+			target.visible_message("<span class='notice'>[M] gives [target] a hug, they look happier!</span>", \
```
Comment (ghost) [inline hippiestation/code/modules/clothing/gloves/softgloves.dm]: You should get rid of this message when the gloves are emagged IMO

Comment (GuyonBroadway) [reply hippiestation/code/modules/clothing/gloves/softgloves.dm]: I could....

**Review by j-mie — APPROVED** [review]

**Comment by GuyonBroadway** [comment]
Jamie says he meant to approve this, not merge it.... I think admins should give +1 or -1 just in case.

**Comment by Arzion5** [comment]
+1

**Pull request merged**